flocker/provision/_install.py

```diff
@@ -212,6 +212,18 @@ def task_install_kernel_devel():
 """)]
 
 
+def task_disable_selinux():
+    """
+    Disable SELINUX for this session and permanently.
+    XXX: Remove this when we work out suitable SELINUX settings.
+    See https://clusterhq.atlassian.net/browse/FLOC-619.
+    """
+    return [
+        Run(command="setenforce 0"),
+        Run(command="test -e /etc/selinux/config && sed --in-place='.preflocker' 's/^SELINUX=.*$/SELINUX=disabled/g' /etc/selinux/config"),
+    ]
+
+
```

Comment:
I don't understand why the docs failed to build. Here's where the new task is imported, so it should be available for use in the `_sphinx.py` module. 
 * https://github.com/ClusterHQ/flocker/pull/1270/files#diff-6a6e7e9f8092167d5a5c2ec85ea37341R9

Documentation builds for me.